lib/Backends/OpenCL/kernels_fwd_conv.cl

```diff
@@ -29,12 +29,25 @@
 // VWM - vector width in dimension M.
 
 static const char* FWD_CONV_CODE = R"(
+#ifndef QUANTIZED
 #define Dtype float
 #define Dtype1 float
 #define Dtype2 float2
 #define Dtype4 float4
 #define Dtype8 float8
 #define Dtype16 float16
+#define ZERO 0.0
+#else
+#define Dtype char
+#define Dtype1 char
+#define Dtype2 char2
+#define Dtype4 char4
+#define Dtype8 char8
+#define Dtype16 char16
+#define ZERO 0
```

Comment:
zeros in quantized version correspond to the offset. Given fp = scale (q - offset), if you want to get fp equal to 0 you should use q == offset.